docker-compose.yaml

```
@@ -11,6 +11,7 @@ services:
       - ./docker/.prebuilt.env
     volumes:
       - postgres-data:/var/lib/postgresql/data
+      - /home/$USER/care_dir-to-read:/backup
```

```suggestion
      - $DB_BACKUPS_DIR:/backup
```
add this variable to .env.example